jetty-start/src/main/java/org/eclipse/jetty/start/Main.java

```diff
@@ -441,7 +441,7 @@ public void start(StartArgs args) throws IOException, InterruptedException
         {
             CommandLineBuilder cmd = args.getMainArgs(args.getDryRunParts());
             cmd.debug();
-            System.out.println(cmd.toQuotedString());
+            System.out.println(cmd);
```

Comment:
This breaks `jetty.sh` usage.
See #9309

I'll try to write a DistributionTest (in this branch) that uses `jetty.sh` (test would be limited to only executing on Linux env)

Reply:
Hmm, this is tricky, as it's a forked Exec (to bash + jetty.sh) that execs a JVM.
Getting the pid to stop the running JVM isn't that straight forward. (nor does `jetty.sh stop` work on this forked/execed JVM)

I'm not going to continue with this testcase, I'm abandoning this effort.

Here's the scenario, in the form of a jetty base, that would cause the issue in #9309

* [requestlog-custom-base.tar.gz](https://github.com/eclipse/jetty.project/files/11958948/requestlog-custom-base.tar.gz)

``` shell
[requestlog-custom-base]$ tree -F
./
├── logs/
├── resources/
│   └── jetty-logging.properties
├── start.d/
│   ├── custom-system-props.ini
│   ├── deploy.ini
│   ├── http.ini
│   └── requestlog.ini
└── webapps/

4 directories, 5 files
[requestlog-custom-base]$ cat start.d/deploy.ini 
--module=deploy
[requestlog-custom-base]$ cat start.d/http.ini 
--module=http
[requestlog-custom-base]$ cat start.d/requestlog.ini 
--module=requestlog
jetty.requestlog.formatString=%{client}a - %u %{dd/MMM/yyyy:HH:mm:ss ZZZ|GMT}t "%r" %s %O "%{Referer}i" "%{User-Agent}i"

[requestlog-custom-base]$ cat start.d/custom-system-props.ini 
--exec
-Dprop="foo bar"
[requestlog-custom-base]$
```